content/graphing/miscellaneous/functions.md

```diff
@@ -299,11 +299,11 @@ For example, an offset of 2 would start graphing at the number 3 ranked series,
 
 ### count_nonzero()
 
-Count all the non-zero values for a given metric
+For a query grouped by some tag key, count the number of tag values with nonzero metric values at each point. E.g. for `count_nonzero(system.load.1{*} by {host})` return a time series representing the number of hosts with nonzero system load at each point.
```

Comment:
The word "some", when used as a determiner, indicates plurality, therefore the object needs to be plural as well. That said, note that the usage here is zero, one, or more tags, which should be expressed clearly. Consider:
> For a query grouped by one or more tag keys, count the […]